src/main/java/org/fundacion/jala/converter/models/parameter/ExtractTextParameter.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ * @author Jorge Caceres Velasco
+ *  * @version 1.0
+ */
+package org.fundacion.jala.converter.models.parameter;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class ExtractTextParameter extends Parameter {
+    private String language;
+    private String resultFile;
+
+    public ExtractTextParameter(final String newFilePath) {
+        super(newFilePath);
+    }
+
+    public ExtractTextParameter(final String newFilePath, final String newLanguage, final String newResultFile) {
+        super(newFilePath);
+        this.language = newLanguage;
+        this.resultFile = newResultFile;
+    }
+
+    public ExtractTextParameter() {
+    }
+    /**
```

Comment:
Specify the Javadoc,  add in third person what does the method do, what type of data is return and what parameter type is needed.

Hope you understand mi comment :).